core/ts.core/src/ts/client/CommandNames.java

```diff
@@ -57,7 +57,9 @@
 
 	// 2.4.0
 	GetApplicableRefactors("getApplicableRefactors", "2.4.0"),
-    GetEditsForRefactor("getEditsForRefactor", "2.4.0");
+    GetEditsForRefactor("getEditsForRefactor", "2.4.0"),
+	
+	OpenExternalProject("openExternalProject");

```

Comment:
Do you know since which version of TypeScript this command is available?

Reply:
Sorry, didn't find it yesterday. It seems old because protocol.d.ts includes it since the first commit 
https://github.com/Microsoft/TypeScript/commits/master/lib/protocol.d.ts

But the history starts at 2016/10 so it's hard to tell